test/test.js

```diff
@@ -370,6 +370,40 @@ describe('sparql-http-client', function () {
 
       return endpoint.constructQuery(simpleConstructQuery, {accept: accept})
     })
+
+    it('should request with default headers', function () {
+      var endpoint = new SparqlHttp({
+        endpointUrl: 'http://example.org/sparql',
+        defaultHeaders: {
+          Authorization: 'Bearer foo'
+        }
+      })
+
+      nock('http://example.org')
+          .get('/sparql?query=' + encodeURIComponent(simpleConstructQuery))
+          .reply(200, function () {
+            assert.equal(this.req.headers.authorization, 'Bearer foo')
+          })
+
+      return endpoint.constructQuery(simpleConstructQuery)
+    })
+
+    it('should override default headers with passed to function', function () {
```

Comment:
```suggestion
    it('should override default headers with argument passed to function', function () {
```
I had a hard time understanding it, I suggest this change